provisioning/e2e/_provisioning_e2e.js

```diff
@@ -80,7 +82,7 @@ describe('IoT Provisioning', function() {
       })
       .then(function() {
         debug('deleting device');
-        registry.delete(deviceId);
+        registry.deleteAsync(deviceId);
```

Comment:
>registry [](start = 8, length = 8)

arent' you supposed to return the promise if you want it to be awaited?

Reply:
YES!  you are my hero.  

---
In reply to: [151319694](https://github.com/Azure/azure-iot-sdk-node/pull/154#discussion_r151319694) [](ancestors = 151319694)